### PR TITLE
Remove default gateway url and allow pull mode only

### PR DESCRIPTION
**Pull request by asdacap**
## Changes:
- Remove default gateway url and allow prometheus pull mode only.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [X] Yes
- [ ] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [X] No

**Comments about testing , should you have some** (optional)

Metrics still work with expose port.

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by asdacap** [comment]
A lot of config also specify localhost for pushgatewayurl. Not sure if I should also remove that.

File: src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg [hunk 1]
```diff
@@ -19,7 +19,6 @@
   "Metrics": {
     "NodeName": "Energy_Web Archive",
     "Enabled": false,
-    "PushGatewayUrl": "http://localhost:9091/metrics",
```
Comment (smartprogrammer93) [inline src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg:22]: Guess we need to do this for all other configs?

**Review by smartprogrammer93 — APPROVED** [review]
LGTM. solves #4697

File: src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg (hunk 1 above)
Comment (asdacap) [reply src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg:22]: Not sure. Probably?

Comment (asdacap) [reply src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg:22]: @LukaszRozmej wanna remove it or not?

Comment (asdacap) [reply src/Nethermind/Nethermind.Runner/configs/energyweb_archive.cfg:22]: I'm gonna just merge this for now.

**Pull request merged**